firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3201,10 +3201,10 @@ extension String {
         value: "Offer to Open Copied Links",
         comment: "Title of setting to enable the Go to Copied URL feature. See https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349")
     public static let SettingsOfferClipboardBarStatus = MZLocalizedString(
-        key: "Settings.OfferClipboardBar.StatusV2",
```

Comment:
The old string with the key `Settings.OfferClipboardBar.StatusV2` should be moved to the `OldStrings` struct (see [here](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#oldstrings-struct)).